CONTRIBUTING.md

```diff
@@ -25,30 +25,49 @@ To run the dev server:
 Actions), please follow these guidelines:
 
-- **One pull request per subject.** Don't combine unrelated changes in a single
-  PR unless they are really subtle details such as fix of a typo.
-
-- **Only PRs into `master` branch are listed in changelog.** PRs into other
-  branches are not picked up by release automation.
-
-- **Name your branches according to nature of change.** Choose one of:
-
-  - `bc/*` for breaking changes
-  - `feature/*` for features
-  - `bugfix/*` for bugfixes
-  - `refactoring/*` for refactoring of the library
-  - `docs/*` or `documentation/*` for changes in docs
-  - `maintenance/*` for maintenance (builds, dependencies, automation, etc.)
-  - `release/*` for [releases](/contribute/releasing) (administrators only)
-
-- **Write clear PR names.** Use imperative and write in English, eg. _Create
-  `ScrollView` component_. Optionally use Markdown code blocks to emphasize.
-  **PR names are used in changelogs.**
-
-- **Write clear commit messages.** Also in imperative and in English. Append
-  issue number (if it exists) in parentheses to the end of the message, eg.
-  _Create `ScrollView` component (#53)_. Optionally use Markdown code blocks to
-  emphasize.
+1. **One pull request per subject.** Don't combine unrelated changes in a single
+   PR unless they are really subtle details such as fix of a typo.
+
+2. **Only PRs into `master` branch are listed in changelog.** PRs into other
+   branches are not picked up by release automation.
+
+3. **Name your branches according to nature of change.** Choose one of:
+
+   - `bc/*` for breaking changes
+   - `feature/*` for features
+   - `bugfix/*` for bugfixes
+   - `refactoring/*` for refactoring of the library
+   - `docs/*` or `documentation/*` for changes in docs
+   - `maintenance/*` for maintenance (builds, dependencies, automation, etc.)
+   - `release/*` for [releases](/contribute/releasing) (administrators only)
+
+4. **Write clear, helpful and descriptive commit messages.**
+
+   1. **Use imperative and write in English,** e.g. _Update dependencies_ or
+      _Claim support for controlled components only_.
+   2. **If an issue exists for your changes, append the issue number** in
+      parentheses to the end of the commit message, e.g. _Update dependencies
+      (#261)_.
+   3. Optionally use Markdown code blocks to emphasize, e.g.
+      _Create `ScrollView` component (#53)_.
+
+5. **Write clear, helpful and descriptive PR names.**
+
+   1. **All rules for commit messages apply** also for PR names.
+   2. **Always check that PR name meets the requirements** above because **PR
+      names are used in changelog**. GitHub automatically truncates long PR
+      names and picks up branch name for multi-commit PRs, so it's necessary to
+      make sure the PR name is what we want to have in the changelog.
+   3. **If an issue exists for your changes, append this text to PR
+      description** in order for the issue to be closed automatically once the
```

Comment:
What is PR Description? I only see title.

Reply:
It's this tiny, easy-to-be-overlooked field (screenshot of new PR screen):

<img width="1292" alt="Snímek obrazovky 2021-09-24 v 19 44 34" src="https://user-images.githubusercontent.com/5614085/134718440-c9d1b8be-e0a9-4739-8ee4-d9f4d1ca9a77.png">

👀

Reply:
They call it _comment_, do you think it's a better name…?

Reply:
Well Im looking at an existing PR and I do not know how to fix it. There is no field called description.

> They call it comment, do you think it's a better name

We should probably use the name GH uses…

Reply:
It's really confusing. The field has no label. If it's empty, it says _No description provided_. In the editing mode, the placeholder says _Leave a comment_ 🤷🏻‍♂️. So I'm using both and let's never talk about it again 🙈.

<img width="950" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/136220330-68e27e25-0813-463f-879f-650708361d41.png">